.ci/teardown.sh

```diff
@@ -31,6 +31,8 @@ if [[ ! $(ps -p 1 | grep systemd) ]]; then
 else
 	echo "Clear Containers Proxy Log:"
 	sudo journalctl --no-pager -u cc-proxy
+	echo "Clear Containers Shim Log:"
+	sudo journalctl --no-pager -t cc-shim
```

Comment:
I have a query/note...
I guess this will show all of the shim log for all of its lifetime - which, my gut tells me we may not want it that contains info from (many) previous runs.... *but*, then my head tells me that this code is currently specifically run in one-shot VMs, so the only log it should have is from this specific test run...
Just noting like - and considering if in the future we might need to take note of the start time of the test run, and then trim the log to include only data from this run, iyswim.

Reply:
Yes. We probably need @chavafg to comment. I personally wipe the journal before running tests locally. Atleast once https://github.com/clearcontainers/shim/pull/90 lands this dump from the beginning-of-time will allow us to atleast link the shim entries with the other pieces.

It's worth noting that once #590 lands, we could be a little cleverer and have the CI call the `cc-log-parser` tool as described on its `README` on teardown so that you'd atleast have a time-ordered dump of the run.

Reply:
I think we can wait until #590 lands and see if that makes debugging easier, if not we can start looking other ideas.